apps/nameguard.io/tailwind.config.ts

```diff
@@ -56,7 +56,7 @@ const config: Config = {
       },
     },
   },
-  plugins: [require("@tailwindcss/forms"), require("tailwind-scrollbar-hide")],
```

Comment:
I'm not sure if we should be removing this yet.

I see that the styling for these checkboxes is broken now:

![CleanShot 2024-09-11 at 15 18 53](https://github.com/user-attachments/assets/6b41dd09-2c59-4208-b943-63d8033ebd6e)

I want us to create standardized checkbox UI components soon, just as we've done for `Button` and now `Input` but we better do that later in a separate PR.